bluesky/commandline/zmq_proxy.py

```diff
@@ -47,17 +46,22 @@ def main():
                         help=("Show 'start' and 'stop' documents. "
                               "(Use -vvv to show all documents.)"))
     parser.add_argument('--logfile', type=str,
-                        help="Write logfile")
+                        help="Redirect logging output to a file on disk.")
     args = parser.parse_args()
     in_port = args.in_port[0]
     out_port = args.out_port[0]
+
     if args.verbose:
-        logger.setLevel('INFO')
-        if args.verbose > 2:
-            logger.setLevel('DEBUG')
+        from bluesky.log import config_bluesky_logging
+        # "INFO" if called with '-v' or '-vv', "DEBUG" if called with '-vvv'
```

Comment:
Would it be enough to have 2 `v`s - `-v` for `INFO` and `-vv` for `DEBUG? I guess `WARNING` will be output by default.

Ref for myself: https://docs.python.org/3/library/logging.html#levels

Reply:
@mrakitin This is how it worked before it was broken.

Reply:
OK, should we revisit it?

Reply:
I think verbosity settings it should be consistent with the rest of our projects. I would vote for `-vvv` for `DEBUG` mode, since it will print a lot of data. Attn: @danielballan

Reply:
>  I would vote for -vvv for DEBUG mode, since it will print a lot of data.

This seems right to me too. I agree that it’s a bit funny that -v and -vv behave the same for now, both here and in caproto CLIs, but that gives us room to add a “medium” level of verbosity as -vv in the future if we want. Looking to programs like ssh and ansible, three or more v’s seems to be where they cross over into extremely high verbosity, a mode too verbose for normal operation, imposing possible performance overhead and generally too much logging to be worth capturing unless you are trying to debug a specific issue.

Reply:
Also the assignment of mode here is artificial. We are not truly select DEBUG mode. Instead the logging mode is used to enable logging of start/stop documents (`logger.info`) and the rest of the documents (`logger.debug`): https://github.com/bluesky/bluesky/blob/b3038a0f80caec4a94f8cf88392962bc44acfaba/bluesky/commandline/zmq_proxy.py#L30
It is also easier to visually locate start/stop documents the logger output in `-vvv` mode.